### PR TITLE
old-web-redirector: rewrite urls to ocis

### PR DESCRIPTION
**Pull request by diocas**
@elizavetaRa @javfg can you review?

Also, @elizavetaRa , do you mind putting a loading spinner in "Redirect.vue" and a prettier message in "Error.vue"? (smth like "this url is not valid. if you thing it should, please contact support")

~~"index2.php" is a workaround since "index.php" is being catched by nginx, but I'll fix that today.~~

### CODE REVIEW

File: old-web-redirector/src/Redirect.vue [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+        default:
+          switch (first) {
+            case '__myshares': {
+              console.log('Redirecting old url to a share')
+              const path = await this.getSharePath(dir, this.$client)
+              this.$router.push({
+                name: path ? 'files-spaces-personal' : 'files-shares-with-me',
+                params: {
+                  storageId: this.user.id,
```
Comment (elizavetaRa) [inline old-web-redirector/src/Redirect.vue:97]: `storageId: 'personal'` when [pr cleaning urls ](https://github.com/cernbox/web/pull/135) gets accepted

**Comment by elizavetaRa** [comment]
@diocas loading spinner + error message sounds good to me

File: old-web-redirector/src/Redirect.vue (hunk 1 above)
Comment (diocas) [reply old-web-redirector/src/Redirect.vue:97]: Yes, I think I added a "TODO" in the code for that

**Comment by diocas** [comment]
@elizavetaRa I've asked if "you" can add those in this PR :)

**Comment by diocas** [comment]
@elizavetaRa I'll merge it for now for testing in qa. We can create a new version after with your spinner.
Also, if @javfg has any comments, we can address them later.

**Pull request merged**